R/install-bitbucket.R

```diff
@@ -115,7 +115,8 @@ remote_package_name.bitbucket_remote <- function(remote, ...) {
 
   bitbucket_DESCRIPTION(
     username = remote$username, repo = remote$repo,
-    host = remote$host, auth = basic_auth(remote))$Package
+    host = remote$host, auth = basic_auth(remote),
+    ref = remote$ref)$Package
```

Comment:
Sg is wrong here...

Reply:
Oh, no,  sorry my bad, forget  it....

Reply:
No worries, I can't remember to use a <- or put spaces in after commas. :)

Reply:
We need the subdirectory here as well, and it would likely be easier to read this if the arguments were in the same order as the formals of `bitbucket_DESCRIPTION`.

```r
bitbucket_DESCRIPTION(
  username = remote$username, repo = remote$repo,
  subdir = remote$subdir, ref = remote$ref,
  host = remote$host, auth = basic_auth(remote)
)$Package
```

Reply:
@jimhester OK, I can do that.

Reply:
done.